linux-cachyos/PKGBUILD

```diff
@@ -215,45 +220,37 @@ fi
 # ZFS support
 if [ -n "$_build_zfs" ]; then
     makedepends+=(git)
-    source+=("git+https://github.com/cachyos/zfs.git#commit=baa50314567afd986a00838f0fa65fdacbd12daf")
+    source+=("git+https://github.com/cachyos/zfs.git#commit=965343f1141aeb38aa7f8bb8f27f82278b87db6e")
 fi
 
 # NVIDIA pre-build module support
 if [ -n "$_build_nvidia" ]; then
     source+=("https://us.download.nvidia.com/XFree86/Linux-x86_64/${_nv_ver}/${_nv_pkg}.run"
-             "${_patchsource}/misc/nvidia/0001-Make-modeset-and-fbdev-default-enabled.patch")
+             "${_patchsource}/misc/nvidia/0001-Make-modeset-and-fbdev-default-enabled.patch"
+             "${_patchsource}/misc/nvidia/0006-nvidia-drm-Set-FOP_UNSIGNED_OFFSET-for-nv_drm_fops.f.patch")
 fi
 
 if [ -n "$_build_nvidia_open" ]; then
     source+=("nvidia-open-${_nv_ver}.tar.gz::https://github.com/NVIDIA/open-gpu-kernel-modules/archive/refs/tags/${_nv_ver}.tar.gz"
              "${_patchsource}/misc/nvidia/0001-Make-modeset-and-fbdev-default-enabled.patch"
              "${_patchsource}/misc/nvidia/0002-Do-not-error-on-unkown-CPU-Type-and-add-Zen5-support.patch"
              "${_patchsource}/misc/nvidia/0003-Add-IBT-Support.patch"
-             "${_patchsource}/misc/nvidia/0006-silence-event-assert-until-570.patch"
-             "${_patchsource}/misc/nvidia/0009-fix-hdmi-names.patch")
+             "${_patchsource}/misc/nvidia/0004-silence-event-assert-until-570.patch"
+             "${_patchsource}/misc/nvidia/0005-nvkms-Sanitize-trim-ELD-product-name-strings.patch"
+             "${_patchsource}/misc/nvidia/0006-nvidia-drm-Set-FOP_UNSIGNED_OFFSET-for-nv_drm_fops.f.patch")
 fi
 
 ## List of CachyOS schedulers
 case "$_cpusched" in
-    cachyos) # CachyOS Scheduler (Sched-ext + BORE + Cachy Sauce)
-        source+=("${_patchsource}/sched/0001-sched-ext.patch"
-                 "${_patchsource}/sched/0001-bore-cachy-ext.patch");;
-    bore) ## BORE Scheduler
+    cachyos|bore) # CachyOS Scheduler (BORE)
         source+=("${_patchsource}/sched/0001-bore-cachy.patch");;
     bmq) ## Project C Scheduler
         source+=("${_patchsource}/sched/0001-prjc-cachy.patch");;
-    eevdf) ## 6.12 EEVDF patches
-        source+=("${_patchsource}/sched/0001-eevdf-next.patch");;
-    rt) ## EEVDF with RT patches
-        source+=("${_patchsource}/misc/0001-rt.patch");;
```

Comment:
Why RT patch was removed?

Reply:
It should be not required anymore, since it got enabled in upstream. If we want to have latest RT features, from the rt tree, then we can keep it, but it also brings more maintenance work

Reply:
The commit of removing RT dependency on CONFIG_EXPERT can also be cherrypicked to, e.g cachy.patch

Reply:
https://git.kernel.org/pub/scm/linux/kernel/git/rt/linux-rt-devel.git/log/?h=linux-6.12.y-rt-rebase This is the latest tree of the rt patchset. Initial thoughts are patch 1-3, 6-9 seem nice to have

Reply:
> It should be not required anymore, since it got enabled in upstream. If we want to have latest RT features, from the rt tree, then we can keep it, but it also brings more maintenance work

From the remaining patches, we don't need the ARM/PowerPC/RISCV specific ones too much since we're not targeting them, but PREEMPT_LAZY and i915 patches are also here. The first one we'll want to have anyway for all kernels, but the second one can be separated into a single patch.

https://gitlab.archlinux.org/archlinux/packaging/upstream/linux-rt/-/tree/linux-6.12.y-rt-patches/patches

Reply:
https://github.com/CachyOS/linux/commits/6.12/rt I've cherrypicked some patches from the rt patchset. This seems to meet the standard expectations of being part of our base patchset. The first two commits is also beneficial for normal kernels, and splitting these into different patches makes no sense in my eyes. The other commits are relatively safe to include for non-RT kernels. 

@ptr1337 what do you think?